cmd/podman/pod_create.go

```diff
@@ -11,6 +12,11 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	// CRI-O default kernel namespaces
```

Comment:
It's not anymore CRI-O only :) I propose something like "Kernel namespaces shared by default within a pod".